build.sbt

```diff
@@ -10,6 +10,14 @@ lazy val root = Project("lila", file("."))
   .aggregate(api)
   .settings(buildSettings)
 
+inThisBuild(
+  List(
+    semanticdbEnabled := true,                       // enable SemanticDB
+    semanticdbVersion := scalafixSemanticdb.revision // use Scalafix compatible version
+  )
+)
+addCompilerPlugin(scalafixSemanticdb)
```

Comment:
this is the [pre-sbt1.3 way of injecting semanticdb](https://scalacenter.github.io/scalafix/docs/users/installation.html#sbt), so it's redundant with the lines above, and it wouldn't work anyway as it needs to be defined on a project-level
```suggestion
```